src/test/java/Homework19.java

```diff
@@ -1,43 +1,58 @@
 import org.openqa.selenium.By;
+import org.openqa.selenium.NoSuchElementException;
 import org.openqa.selenium.WebElement;
 import org.testng.Assert;
+import org.testng.annotations.Optional;
+import org.testng.annotations.Parameters;
 import org.testng.annotations.Test;
 
 public class Homework19 extends BaseTest {
 
     @Test
-    public void deletePlaylist() throws InterruptedException {
+    @Parameters({"baseURL"})
+    public void deletePlaylist(@Optional String baseURL) throws InterruptedException {
```

Comment:
**Needless Repetition!**  You can delete these variables because they are in the 
`class BaseTest` 

![image](https://user-images.githubusercontent.com/104761482/200297412-4d516bd7-6884-4b7f-8d70-b678bc6a7527.png)